### DISCUSSION TITLE
How do I insert a base64 image into the editor?

### DISCUSSION THREAD

**Discussion by PineSongCN** [personalizedrefrigerator/js-draw#83] [Q&A] [answered]
How do I insert a base64 image into the editor?

**Comment by personalizedrefrigerator** [comment]
See the example for [Editor.toDataURL](https://js-draw.web.app/typedoc/classes/js_draw.Editor.html#toDataURL).

**Reply by PineSongCN** [reply]
> See the example for [Editor.toDataURL](https://js-draw.web.app/typedoc/classes/js_draw.Editor.html#toDataURL).

Thank you. It's import，not export.

**Reply by personalizedrefrigerator** [reply, marked as answer]
To clarify, the example on the Editor.toDataURL page shows how to:
- Load a PNG image from a data URL and add it to the editor (using `ImageComponent`).
- Add a toolbar button that:
   - Converts the editor to a base64 data URL.
   - Adds that as an image to the editor.

**Reply by PineSongCN** [reply]
> To clarify, the example on the Editor.toDataURL page shows how to:
> 
> * Load a PNG image from a data URL and add it to the editor (using `ImageComponent`).
> * Add a toolbar button that:
>   
>   * Converts the editor to a base64 data URL.
>   * Adds that as an image to the editor.

Thank you.
